tests/utils_tests/test_safestring.py

```diff
@@ -7,6 +7,8 @@
 
 
 class customescape(str):
+    __slots__ = ()
```

Comment:
Changes is tests prove that it is another auto-generated PR. There is no need for micro-optimizations in tests.